src/components/editor/InputLookupQA.jsx

```diff
@@ -72,106 +72,111 @@ class InputLookupQA extends Component {
     }
 
     render() {
+      let isMandatory, isRepeatable, authority, subauthority, language
+      const lookupConfigs = this.props.lookupConfig
+
+      try {
+        isMandatory = JSON.parse(this.props.propertyTemplate.mandatory)
+      } catch (error) {
+        //We do not really care about this error. If there is an error the variable will be undefined and hence false
+      }
+
+      try {
+        isRepeatable = JSON.parse(this.props.propertyTemplate.valueConstraint.repeatable)
+      } catch (error) {
+        //We do not really care about this error. If there is an error the variable will be undefined and hence false
+      }
+
+    const typeaheadProps = {
+        required: isMandatory,
+        multiple: isRepeatable,
+        placeholder: this.props.propertyTemplate.propertyLabel,
+        useCache: true,
+        isLoading: this.state.isLoading,
+        options: this.state.options,
+        selected: this.state.selected,
+        delay: 300
+    }
 
-        let isMandatory, isRepeatable, authority, subauthority, language
-        const lookupConfigs = this.props.lookupConfig
-
-        try {
-            isMandatory = JSON.parse( this.props.propertyTemplate.mandatory )
-            isRepeatable = JSON.parse( this.props.propertyTemplate.repeatable )
-        } catch ( error ) {
-            console.error( `Problem with properties fetched from resource template: ${error}` )
-        }
-        const typeaheadProps = {
-            required: isMandatory,
-            multiple: isRepeatable,
-            placeholder: this.props.propertyTemplate.propertyLabel,
-            useCache: true,
-            isLoading: this.state.isLoading,
-            options: this.state.options,
-            selected: this.state.selected,
-            delay: 300
-        }
-
-        return (
-            <div>
-                <AsyncTypeahead id="lookupComponent"
+    return (
```

Comment:
I'm guessing this is going to be... annoying to rebase on `master` now. Let me know if I can do anything to help.

Reply:
I'm afraid to find out what you mean....